install/kubernetes/cilium/values.yaml

```diff
@@ -358,6 +358,11 @@ containerRuntime:
 
 # crdWaitTimeout: ""
 
+# -- Tail call hooks for custom eBPF programs.
+customCalls:
+  # -- Enable tail call hooks for custom eBPF programs.
+  enable: false
```

Comment:
Shouldn't this be `enabled`?

We also need to have a discussion at some point about `feature=true` vs. `feature.enabled=true`. Right now we're mixing both :S

Reply:
in general, it should be `feature.enabled=true` otherwise there's no way to extend the configuration of that outer object.